framework/src/packets/icmp/v6/ndp/options/mod.rs

```diff
@@ -26,6 +27,12 @@ pub enum NdpOption {
     Undefined(u8, u8),
 }
 
+pub trait NdpAddOptions {
```

Comment:
more a q, should this be a trait on ndppacket, since the getter this there?

Reply:
added the method on the trait